### PR TITLE
[ci] put primary condition to rector github action run

### PR DESCRIPTION
**Pull request by TomasVotruba**
@samsonasik I had this in mind :) suggested by ChatGPT

Could you test it on the use case that was failing?

### CODE REVIEW

File: .github/workflows/rector.yaml [hunk 1]
```diff
@@ -43,15 +35,11 @@ jobs:
 
             -   run: composer install --no-progress --ansi
 
-            ## First run Rector - here can't be --dry-run !!! it would stop the job with it and not commit anything in the future
+            ## here can't be --dry-run, as the change results in error code 1 and stops the job without commiting in the future
             -   run: bin/rector process ${{ matrix.paths }} --ansi
 
-            -   run: vendor/bin/ecs check --fix --ansi
```
Comment (samsonasik) [inline .github/workflows/rector.yaml:49]: cs fix still needed after apply rector

**Comment by samsonasik** [comment]
Also, rector workflow is pending with this

```
 rector (config utils) Expected — Waiting for status to be reported 
```

**Comment by samsonasik** [comment]
I will create separate alternative PR

File: .github/workflows/rector.yaml (hunk 1 above)
Comment (TomasVotruba) [reply .github/workflows/rector.yaml:49]: This is to comply with dependency-less defaults, to make it easier to spread and use our own command to generate it on the fly :)
We'll use the coding standard in standalone run.

**Comment by TomasVotruba** [comment]
> Also, rector workflow is pending with this

Hm, that looks like some bug. The condition is possibly using wrong string :thinking:

**Pull request closed without merging**